SquareInAppPaymentsSDK.podspec

```diff
@@ -30,4 +30,10 @@ Pod::Spec.new do |s|
   s.source_files = 'SquareInAppPaymentsSDK.framework/**/*.{h,hh}'
   s.public_header_files = 'SquareInAppPaymentsSDK.framework/Headers/*.{h,hh}'
   s.vendored_frameworks = 'SquareInAppPaymentsSDK.framework'
+  s.script_phase = {
+      name: 'Remove Square In-App Payments SDK setup script for Cocoapod installs',
+      # Only remove the setup script if the host app does not contain a build phase that executes it.
```

Comment:
# Only remove the setup script if the host app does not contain a build phase that executes it `or the project file cannot be found.`